app-office/fcrm/patches/fcrm-0.2.0~beta.patchset

```diff
@@ -0,0 +1,26 @@
+From 2be60306aeca28d22bcb71727de6fb747e400171 Mon Sep 17 00:00:00 2001
+From: Mazbrili <mazbrili@yahoo.com>
+Date: Sat, 24 Aug 2019 07:27:34 +0700
+Subject: add haiku-specific location
+
+
+diff --git a/src/main.cpp b/src/main.cpp
+index 63212c0..977a4e1 100644
+--- a/src/main.cpp
++++ b/src/main.cpp
+@@ -13,7 +13,11 @@ void initSettings() {
+ 
+     qDebug() << "Settings are in " << settings.fileName();
+ 
+-    auto data_path = QStandardPaths::writableLocation(QStandardPaths::AppDataLocation);
++#ifdef Q_OS_HAIKU
++  auto data_path = QStandardPaths::writableLocation(QStandardPaths::HomeLocation)+"/config/settings/TheLastViking/F-crm";
```

Comment:
This should not be needed.

It appears the Qt recipe is patched to have AppDataLocation be in cache. This is wrong. Why was this done; cc @threedeyes?

Reply:
@waddlesplash  so i need waiting confirmation from @threedeyes  about it or... just drop my pachset anyway?

Reply:
What should be the correct place?

Reply:
AppDataLocation should return this path already, right? It shouldn't need individual patching per app.

Reply:
patch deleted